ui/src/table.tsx

```diff
@@ -483,6 +531,96 @@ const
         {colContextMenuList && <Fluent.ContextualMenu {...colContextMenuList} />}
       </>
     )
+  }),
+  Pagination = ({ currentPage, onPageChange, pagination }: PaginationProps) => {
+    const
+      { total_rows, rows_per_page } = pagination,
+      lastPage = Math.ceil(total_rows / rows_per_page),
+      btnStyles: Fluent.IButtonStyles = { rootDisabled: { background: 'transparent' }, root: { marginLeft: -8 } },
+      isLastPage = currentPage === lastPage
```

Comment:
```suggestion
      isLastPage = currentPage === lastPage || lastPage === 0
```
Pagination controls are not disabled in case no search results are found:
https://user-images.githubusercontent.com/23740173/161576060-e95cf06f-171a-478d-9013-e0d13b6ba9e4.mov

Reply:
Good catch!